public/app/modules/home/home.html

```diff
@@ -10,129 +10,103 @@
 	<div class="row">
 		<div class="col-sm-12 col-md-4 col-lg-4">
 			<div class="row">
-				<div class="col-sm-6 col-md-12 col-lg-12">
-					<div class="box box-4col box-1" style="background-image: url(/images/post_its.jpg);">
-						<div class="box-caption">
-							<div class="label label-text-2 black-text">IDEAS</div>
+				<a ng-href="#/{{page_content[0].link}}" ng-if="page_content[0]">
+					<div class="col-sm-6 col-md-12 col-lg-12">
+						<div class="box box-4col box-1" style="background-image: url({{page_content[0].image.secure_url}});">
+							<div class="box-caption">
+								<div class="label label-text-2 black-text">IDEAS</div>
                                     <span class="title-2 white-text">
-                                        A Caribbean Open Data Public Service
+                                        {{page_content[0].title}}
                                     </span>
+							</div>
 						</div>
 					</div>
-				</div>
-				<div class="col-sm-6 col-md-12 col-lg-12">
-					<div class="box box-4col box-2" style="background-image: url(/images/devca.jpg);">
-						<div class="box-caption">
-							<div class="label label-text-2 black-text">IDEAS</div>
-                                    <span class="title-2 white-text">
-                                        A Caribbean Open Data Public Service
-                                    </span>
+				</a>
+				<a ng-href="#/{{page_content[1].link}}" ng-if="page_content[1]">
```

Comment:
@tremainebuchanan format the data in `page_content` so that you don't have to statically reference items.